src/Distributions.jl

```diff
@@ -230,6 +234,7 @@ export
     rate,               # get the rate parameter
     sampler,            # create a Sampler object for efficient samples
     scale,              # get the scale parameter
+    scale!,             # provide storage for the scale parameter (used in multivariate distribution mvlognormal)
```

Comment:
But if you don't feel so, don't listen to me – I'm not an expert on this domain.